cmake/cvars.cmake

```
@@ -0,0 +1,50 @@
+set(CVAR_VSYNC_ENABLED "gVsyncEnabled" CACHE STRING "")
+set(CVAR_Z_FIGHTING_MODE "gZFightingMode" CACHE STRING "")
+set(CVAR_NEW_FILE_DROPPED "gNewFileDropped" CACHE STRING "")
+set(CVAR_DROPPED_FILE "gDroppedFile" CACHE STRING "")
+set(CVAR_INTERNAL_RESOLUTION "gInternalResolution" CACHE STRING "")
+set(CVAR_MSAA_VALUE "gMSAAValue" CACHE STRING "")
+set(CVAR_SDL_WINDOWED_FULLSCREEN "gSdlWindowedFullscreen" CACHE STRING "")
+set(CVAR_TEXTURE_FILTER "gTextureFilter" CACHE STRING "")
+set(CVAR_CONTROL_NAV "gControlNav" CACHE STRING "")
+set(CVAR_CONSOLE_ENABLED "gConsoleEnabled" CACHE STRING "")
+set(CVAR_CONTROLLER_CONFIGURATION_ENABLED "gControllerConfigurationEnabled" CACHE STRING "")
+set(CVAR_CONTROLLER_DISCONNECTED_ENABLED "gControllerDisconnectedWindowEnabled" CACHE STRING "")
+set(CVAR_CONTROLLER_REORDERING_ENABLED "gControllerReorderingWindowEnabled" CACHE STRING "")
+set(CVAR_GFX_DEBUGGER_ENABLED "gGfxDebuggerEnabled" CACHE STRING "")
+set(CVAR_STATS_ENABLED "gStatsEnabled" CACHE STRING "")
+set(CVAR_ENABLE_MULTI_VIEWPORTS "gEnableMultiViewports" CACHE STRING "")
+set(CVAR_LOW_RES_MODE "gLowResMode" CACHE STRING "")
+set(CVAR_SIMULATED_INPUT_LAG "gSimulatedInputLag" CACHE STRING "")
+set(CVAR_ALT_ASSETS "gAltAssets" CACHE STRING "")
+set(CVAR_SINK_ENABLED "gSinkEnabled" CACHE STRING "")
+set(CVAR_GAME_OVERLAY_FONT "gOverlayFont" CACHE STRING "")
+set(CVAR_OPEN_MENU_BAR "gOpenMenuBar" CACHE STRING "")
+set(CVAR_BLOCK_CONTROLLERS "gControllers" CACHE STRING "")
+set(CVAR_BLOCK_ADVANCED_RESOLUTION "gAdvancedResolution" CACHE STRING "")
```

using "block" made me think this was a cvar that would *block access* to these or something, i think

```suggestion
set(CVAR_PREFIX_CONTROLLERS "gControllers" CACHE STRING "")
set(CVAR_PREFIX_ADVANCED_RESOLUTION "gAdvancedResolution" CACHE STRING "")
```

might be less confusing

---

That was originally your name suggestion XD